content/articles/factors_that_affects_the_performance_of_system_using_deep_learning/index.md

```diff
@@ -0,0 +1,92 @@
+ ######  Basic problems
+* What kind of system should be built? 
+* How will the developed algorithm handle the immediate user’s demands?
+* How easily will re-engineering be made possible?
+
+#### ii. Evaluation difficulties of default hyperparameters for multiple optimisers.
+          Poor accuracy is obtained here as a result of low awareness from training the system data sets. The Adams optimiser is used in this deep system algorithm, which is adaptable. 
+
+#### iii. Technical expertise in deep learning system network design.
+
+          The demand and need for deep learning experts is highly recommended here. Experience is needed to increase the learning rate and improve the system's performance.
+
+#### iv. Which optimiser suits the deep network best?
+          According to research, the Adam optimiser offers the highest and best performance rate. The ability to outperform other approaches (optimisers) is lacking. 
+
+### How To Get A Perfect Performance Rate From A Deep Learning System
+
+#### 1. A Deep Neural Network System's Individual Units' (Layers') Functionality 
+
+Convolutional neural networks must take individual activation levels into account. This step, though, is difficult to achieve. First, the issue of unit interpretability in deep networks must be grasped.
+This knowledge will assist the network in avoiding becoming a completely opaque black box. Our measurement method is network dissection. This method calculates a system's deep learning performance rate.
+Measurements of alignments between different levels are also part of network dissection in the deep convolutional network (units). It is being done to separate dense data sets for a quicker performance rate. This segmentation notion is also known as "broden." Other classifier networks can use the Broden approach as well. The Progressive generative adversarial network with the VGG-16 scene classifier (P-GAN). The most common are these two.
+The network's training methods are the generator and discriminator progression layers. A convolutional neural network (CNN)-based classifier is made up of two layers. These layers, which are referred to as the earlier and later layers,  The final layer is made up of two levels: objects and parts. The system associates with the layers and emerges when it notices this sub-layer. There is a lot of color dictation in the early levels.
+ Research has highlighted the importance of performance rate categorization in CNN neurons. Some deep neural networks are trained to create scenes, while others are trained to recognize them. This helps to improve the data flow in the system. 
+Manipulation of GAN's unit-producing outlets is complicated by artificial pruning. Trained images, like adversarial examples, are developed, categorized, and highlighted on certain neurons. These neurons are treated in some way in order to attain precision. 
```

Comment:
Why are these points more aligned towards CNN or GAN? Shouldn't it be generalized?

Explanation of various terminologies are missing